dmapiclient/data.py

```diff
@@ -866,17 +870,21 @@ def get_direct_award_project_search(self, user_id, project_id, search_id):
             }
         )
 
-    def get_direct_award_project_services(self, user_id, project_id, fields=[]):
+    def get_direct_award_project_services(self, project_id, user_id=None, fields=[]):
+        params = {"user-id": user_id}
+        if fields:
+            params.update({"fields": ','.join(fields)})
```

Comment:
Seems a little verbose over `params["fields"] = ",".join(fields)` but unimportant

Reply:
`params["fields"] = ",".join(fields)` would set the `fields` param to `''` if it was `[]`

Reply:
I wasn't suggesting you remove the `if fields:`